test/controllers/user.js

```diff
@@ -195,4 +196,117 @@ describe('/POST users/sessions', () => {
 });
 
 
+/*
+* Testing the /admin/users (POST) route
+*/
+describe('/POST admin/users', () => {
+
+  const admin = {
+    name: 'Kevin',
+    lastName: 'Temes',
+    email: 'kevin.temes@wolox.com.ar',
+    password: '12345678',
+    isAdmin: true
+  };
+
+  const notAnAdmin = {
+    name: 'Not',
+    lastName: 'AnAdmin',
+    email: 'not.an.admin@wolox.com',
+    password: '12345678'
+  };
+
+  const newAdmin = {
+    name: 'Admin',
+    lastName: 'Admin',
+    email: 'admin@wolox.com',
+    password: '12345678'
+  };
+
+  const adminLogin = {
+    email: 'kevin.temes@wolox.com.ar',
+    password: '12345678'
+  };
+
+  const notAdminLogin = {
+    email: 'not.an.admin@wolox.com',
+    password: '12345678'
+  };
+
+  it('should successfully create an admin user', (done) => {
+
+    User.create(admin).then(res => {
+      chai.request(server)
+        .post('/users/sessions')
+        .send(adminLogin)
+        .then((res) => {
+          chai.request(server)
+            .post('/admin/users')
+            .send(newAdmin)
+            .set('token', res.body.token)
+            .then(result => {
+              result.should.have.status(201);
+              result.body.should.include({name: 'Admin', lastName: 'Admin', email: 'admin@wolox.com'});
+              done();
+            });
+        });
+    });
+
+  });
+
+  it('should successfully update a regular user to an admin user', (done) => {
+
+    User.create(admin).then(res => {
+      User.create(notAnAdmin).then(res => {
+        chai.request(server)
+          .post('/users/sessions')
+          .send(adminLogin)
+          .then((res) => {
+            chai.request(server)
+              .post('/admin/users')
+              .send(notAnAdmin)
+              .set('token', res.body.token)
+              .then(result => {
+                result.should.have.status(201);
+                User.findOne({ where: {email: notAnAdmin.email} }).then(user => {
+                  assert(user.isAdmin);
```

Comment:
Why are you using assert instead of should? We should try to be consistent inside the same proyect and use the same tool when possible, otherwise it will start to grow into a huge mess.